src/Data/Managers/V2/helpers/utils.ts

```diff
@@ -0,0 +1,11 @@
+export function urlEncodeParams<Type extends object>(params: Type) {
```

Comment:
docstrings. 
File name could be the same as the method considering it only contains that.